### DISCUSSION TITLE
post_233844

### DISCUSSION THREAD

**Discussion by cincuranet** [cincuranet/blog-comments#1348] [Comments]
imported 

https://www.tabsoverspaces.com/id/233844

**Comment by cincuranet** [comment]
Imported comment written by _svick_ on _2020-11-14T05:56:29_:

<p>The part I would stress is that `A` is a getter-only *auto-property*. That's important, because "auto-property" means that that it has a hidden backing field, as you show in the last snippet.</p>

**Comment by ATGeorge** [comment]
2 years later, but thanks for this explanation!  I couldn't find much on this, but had a sneaking suspicion that my use of "=> new..." on a property would cause a ton of wasted space.  Not only did your post confirm that, but gave the syntax for the behavior I needed!
